topology/probes/libvirt/libvirtgo.go

```diff
@@ -0,0 +1,141 @@
+// +build libvirt,linux
+
+/*
+ * Copyright (C) 2018 Orange.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy ofthe License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specificlanguage governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package libvirt
+
+import (
+	"context"
+	"fmt"
+
+	libvirtgo "github.com/libvirt/libvirt-go"
```

Comment:
not consistent binding either `libvirtgo` or `libvirt` used consistently throughout the project